ts/packages/node/src/index.ts

```diff
@@ -11,6 +11,8 @@ export * from './stores/HashiCorpVault';
 export * from './stores/Keyv';
 export * from './stores/KubernetesSecret';
 export * from './stores/LaunchDarkly';
```

Comment:
No need for the open feature prefix just set those the main launchdarklyconfigstore same for all others. Same for filenames and types. Remove the old one.

Reply:
Sure. this means breaking changes for any `LaunchDarklyConfigStore` since store.set() is disabled.
we're ok with that?